airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -673,9 +673,8 @@ def check_status(
             non_terminal_states = self.non_terminal_states
 
         sec = 0
-        running = True
 
-        while running:
+        while True:
```

Comment:
had to do this change otherwise python would complain that the `response` variable might be uninitialized in the return clause.